test/default-xpring-client-test.ts

```diff
@@ -436,17 +454,52 @@ describe('Default Xpring Client', function(): void {
     assert.equal(exists, true)
   })
 
-  it('Check if account exists - failing network request', async function() {
-    // GIVEN a XpringClient which wraps an erroring network client.
+  it('Check if account exists - failing network request w/ NOT_FOUND error', async function() {
+    // GIVEN a XpringClient which wraps an erroring network client that returns an Error containing
+    // a property 'code' with value grpc.status.NOT_FOUND.
+    const errorWithCode5 = new FakeServiceError(
```

Comment:
I would call this `notFoundError`, since `5` is kind of a magic number and this isn't a really descriptive name.

Reply:
way better